internal/transform/contract_code.go

```diff
@@ -26,6 +27,11 @@ func TransformContractCode(ledgerChange ingest.Change, header xdr.LedgerHeaderHi
 		return ContractCodeOutput{}, nil
 	}
 
+	ledgerKey, _ := ledgerEntry.LedgerKey()
+	ledgerKeyByte, _ := ledgerKey.MarshalBinary()
+	hashedLedgerKeyByte := hash.Hash(ledgerKeyByte)
+	ledgerKeyHash, _ := strkey.Encode(strkey.VersionByteContract, hashedLedgerKeyByte[:])
+
```

Comment:
Not a big deal, but might be interesting to encapsulate into a function if there are other instances in the future where we need to return `ledgerKeyHash`

Reply:
Oh yeah I should make this a function